cmd/manager/main.go

```diff
@@ -78,7 +78,7 @@ type managerConfig struct {
 	ProbeAddr             string `envconfig:"HEALTH_PROBE_BIND_ADDR" default:":8081"`
 	EnvoyControlPlaneAddr string `envconfig:"ENVOY_CONTROL_PLANE_BIND_ADDR" default:":18000"`
 	EnableLeaderElection  bool   `envconfig:"ENABLE_LEADER_ELECTION" default:"false"`
-	LogLevel              string `envconfig:"LOG_LEVEL" default:"INFO"`
+	LogLevel              string `envconfig:"LOG_LEVEL" default:"DEBUG"`
```

Comment:
Can we revert this back to INFO? I think DEBUG for the default logging level is too verbose